docs: Serverless/Dedicated deployment tiers, scale-to-zero, and usage-based pricing (#4815)
langchain-ai/docs: src/langsmith/scalability-and-resilience.mdx

## Postgres resilience

> For deployment modalities where we manage the Postgres database, we have periodic backups and continuously replicated standby replicas for automatic failover. This Postgres configuration is available in the [Cloud deployment option](/langsmith/cloud) for [`Dedicated` deployment types](/langsmith/cloud-platform-features#deployment-types) only.

Reviewer: ```suggestion
For deployment modalities where LangSmith manages the Postgres database, there are periodic backups and continuously replicated standby replicas for automatic failover. This Postgres configuration is available in the [Cloud deployment option](/langsmith/cloud) for [Dedicated deployment type](/langsmith/cloud-platform-features#deployment-types) only.
```
